cli/src/commands/router/commands/compose.ts

```diff
@@ -139,7 +139,7 @@ export default (opts: BaseCommandOptions) => {
       }),
     });
 
-    routerConfig.version = createHash('sha1').update(routerConfig.toJsonString()).digest('hex');
+    routerConfig.version = randomUUID();
```

Comment:
This has the effect that the server is always swapped even when the config hasn't changed. I think this is fine but we should also check in the controlplane if version is valid UUID before we use it in the query to avoid such errors in the future.

Reply:
Done